### DISCUSSION TITLE
How to Make tsParticles Confetti (Falling, Image) Work

### DISCUSSION THREAD

**Discussion by shanekryme** [matteobruni/tsparticles#5359] [Q&A]
Hi, Seeking some assistance with getting tsParticles Confetti (Falling, Image) to work on my WordPress website.

Trying to load the following but cant seem to get it to work on my website.

https://confetti.js.org/

**Cpnfigs** > Falling
**Shapes** > Image

**Toggle Code**

Also included the following.
<script src="https://cdn.jsdelivr.net/npm/@tsparticles/confetti@3.0.3/tsparticles.confetti.bundle.min.js"></script>

Is there something I am missing to get this to work?

**Comment by matteobruni** [comment]
Can you include more details? Like the code?

**Reply by shanekryme** [reply]
The code is at the links listed above. I am trying to copy the code to put on my website however it isn't working.

From that link above I am trying to use the **Falling Confetti** with the **shapes** being **images**.

https://confetti.js.org/

Configs > Falling
Shapes > Image

Toggle Code

Also included the following below based on the instructions from https://confetti.js.org/more.html

<script src="https://cdn.jsdelivr.net/npm/@tsparticles/confetti@3.0.3/tsparticles.confetti.bundle.min.js"></script>

**Reply by shanekryme** [reply]
The Code is as follows:

`tsParticles.load({
          id: "tsparticles",
          options: {
  "fullScreen": {
    "zIndex": 1
  },
  "particles": {
    "color": {
      "value": [
        "#FFFFFF",
        "#FFd700"
      ]
    },
    "move": {
      "direction": "bottom",
      "enable": true,
      "outModes": {
        "default": "out"
      },
      "size": true,
      "speed": {
        "min": 1,
        "max": 3
      }
    },
    "number": {
      "value": 500,
      "density": {
        "enable": true,
        "area": 800
      }
    },
    "opacity": {
      "value": 1,
      "animation": {
        "enable": false,
        "startValue": "max",
        "destroy": "min",
        "speed": 0.3,
        "sync": true
      }
    },
    "rotate": {
      "value": {
        "min": 0,
        "max": 360
      },
      "direction": "random",
      "move": true,
      "animation": {
        "enable": true,
        "speed": 60
      }
    },
    "tilt": {
      "direction": "random",
      "enable": true,
      "move": true,
      "value": {
        "min": 0,
        "max": 360
      },
      "animation": {
        "enable": true,
        "speed": 60
      }
    },
    "shape": {
      "type": "image",
      "options": {
        "image": [
          {
            "src": "https://particles.js.org/images/fruits/apple.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/avocado.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/banana.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/berries.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/cherry.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/grapes.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/lemon.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/orange.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/peach.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/pear.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/pepper.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/plum.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/star.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/strawberry.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/watermelon.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          },
          {
            "src": "https://particles.js.org/images/fruits/watermelon_slice.png",
            "width": 32,
            "height": 32,
            "particles": {
              "size": {
                "value": 16
              }
            }
          }
        ]
      }
    },
    "size": {
      "value": {
        "min": 2,
        "max": 4
      }
    },
    "roll": {
      "darken": {
        "enable": true,
        "value": 30
      },
      "enlighten": {
        "enable": true,
        "value": 30
      },
      "enable": true,
      "speed": {
        "min": 15,
        "max": 25
      }
    },
    "wobble": {
      "distance": 30,
      "enable": true,
      "move": true,
      "speed": {
        "min": -15,
        "max": 15
      }
    }
  }
}
      });`

With 
<script src="https://cdn.jsdelivr.net/npm/@tsparticles/confetti@3.0.3/tsparticles.confetti.bundle.min.js"></script>

Included in the page.
